staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5006,7 +5006,7 @@ type Sysctl struct {
 	// Value of a property to set
 	Value string `protobuf:"bytes,2,opt,name=value"`
 	//
-	Unsafe bool
+	Unsafe bool `protobuf:"varint,3,opt,name=unsafe"`
```

Comment:
all the fields in this struct should also have json tags

Reply:
json tags added